frontend/app/src/App.tsx

```diff
@@ -1598,6 +1605,8 @@ export class App extends PureComponent<Props, State> {
             setTheme: this.setAndSendTheme,
             availableThemes: this.props.theme.availableThemes,
             addThemes: this.props.theme.addThemes,
+            hideFullScreenButton,
```

Comment:
This might be dumb. I don't know how to test this, but is there a way we know the App does not accidentally sets this to true for Streamlit proper? Perhaps @mayagbarnes may know the solution with RTL (it kinda violates the philosophy, but I am trying to avoid an accidental checkin of setting it to true.

Reply:
I can just add a test that makes sure that it defaults to `false`

Reply:
Not anything off the top of my head. We would have to just add another (Enzyme) test to App to check the state.

I think it would be enough to test that the `FullScreenWrapper` renders the correct elements (StyledFullScreenButton) normally, and then manually set the context value - via the new util render function Will made - to the `true` (probably best separated in a distinct describe block) to test that it does not render anymore.
More comprehensively, we should probably add a test for both options in each element that uses the `FullScreenWrapper` but that can be a follow up.